OpenRA.Mods.Common/Widgets/ConfirmationDialogs.cs

```diff
@@ -37,6 +49,21 @@ public static void PromptConfirmAction(string title, string text, Action onConfi
 				if (onCancel != null)
 					onCancel();
 			};
+
+			if (!string.IsNullOrEmpty(restartText) && restartButton != null)
+			{
+				restartButton.IsVisible = () => true;
```

Comment:
It would make more sense to toggle the visibility (for all three buttons) based on whether the action is null or not.